src/lsm/table.zig

```diff
@@ -759,13 +786,17 @@ pub fn TableType(
             );
         }
 
-        pub inline fn data_block_values_used(data_block: BlockPtr) []const Value {
+        pub inline fn data_block_values_used(data_block: BlockPtrConst) []const Value {
             const header = mem.bytesAsValue(vsr.Header, data_block[0..@sizeOf(vsr.Header)]);
             // TODO we should be able to cross-check this with the header size
             // for more safety.
             const used = @intCast(u32, header.request);
             assert(used <= data.value_count_max);
-            return data_block_values(data_block)[0..used];
+            const values = mem.bytesAsSlice(
+                Value,
+                data_block[data.values_offset..][0..data.values_size],
+            );
+            return values[0..used];
```

Comment:
`data_block_values()` takes a mutable pointer so it can't be used here.

Perhaps I should have called this function `data_block_values_used_const()` but we don't have any need for mutable version currently...

Reply:
We could also use `const slice` here like the others.

Reply:
> We could also use `const slice` here like the others.

done!